FloconDesktop/data/remote/src/commonMain/kotlin/com/flocon/data/remote/dashboard/models/DashboardConfigDataModel.kt

```diff
@@ -1,19 +1,53 @@
 package com.flocon.data.remote.dashboard.models
 
+import io.github.openflocon.domain.dashboard.models.ContainerConfigDomainModel
+import io.github.openflocon.domain.dashboard.models.FormContainerConfigDomainModel
+import io.github.openflocon.domain.dashboard.models.SectionContainerConfigDomainModel
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.SerialName
 import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.JsonClassDiscriminator
 
 @Serializable
 data class DashboardConfigDataModel(
     val dashboardId: String,
-    val sections: List<SectionConfigDataModel>,
+    val containers: List<DashboardContainerDataModel>,
 )
 
 @Serializable
-data class SectionConfigDataModel(
+data class DashboardContainerDataModel(
     val name: String,
     val elements: List<DashboardElementDataModel>,
+    val containerConfig: ContainerConfigDataModel
 )
 
+@OptIn(ExperimentalSerializationApi::class)
+@JsonClassDiscriminator("containerType")
+@Serializable
+sealed class ContainerConfigDataModel {
+    abstract fun unwrap(): ContainerConfigDomainModel
+}
+
+@Serializable
+@SerialName("FORM")
+data class FormContainerConfigDataModel(
+    val formId: String,
+    val submitText: String,
+) : ContainerConfigDataModel() {
+    override fun unwrap(): ContainerConfigDomainModel =
```

Comment:
instead of a unwrap, in the project we use extension methods mappers, easier to read & find and not part of the data model